cmd/gateway-azure.go

```diff
@@ -125,14 +126,19 @@ func azureToObjectError(err error, params ...string) error {
 }
 
 // Inits azure blob storage client and returns AzureObjects.
-func newAzureLayer(args []string) (GatewayLayer, error) {
-	endPoint, secure, err := parseGatewayEndpoint(args[0])
-	if err != nil {
-		return nil, err
-	}
+func newAzureLayer(args cli.Args) (GatewayLayer, error) {
+
+	var err error
 
-	if endPoint == "" {
-		endPoint = storage.DefaultBaseURL
+	// Default endpoint parameters
+	endPoint := storage.DefaultBaseURL
+	secure := true
+
+	if len(args) > 0 {
```

Comment:
There is no need to do this.. it won't crash. args.Get(0) is safe, returns empty if not found.  In-fact just use `args.First()`